node_app/client/index.html

```diff
@@ -186,31 +102,121 @@ <h4 class="interTitle">Where you can use it</h4>
+                        </svg>
+                    </div>
+                    <div>
+                        <p> Media Project</p>
+                        <svg xmlns="http://www.w3.org/2000/svg" width="128" height="128" fill="currentColor"
+                            class="bi bi-play-btn" viewBox="0 0 16 16">
+                            <path stroke="white" stroke-width="0.5px"
+                                d="M6.79 5.093A.5.5 0 0 0 6 5.5v5a.5.5 0 0 0 .79.407l3.5-2.5a.5.5 0 0 0 0-.814l-3.5-2.5z" />
+                            <path stroke="white" stroke-width="0.5px"
+                                d="M0 4a2 2 0 0 1 2-2h12a2 2 0 0 1 2 2v8a2 2 0 0 1-2 2H2a2 2 0 0 1-2-2V4zm15 0a1 1 0 0 0-1-1H2a1 1 0 0 0-1 1v8a1 1 0 0 0 1 1h12a1 1 0 0 0 1-1V4z" />
+                        </svg>
+                    </div>
+                    <div>
+                        <p> Personal Reasons</p>
+                        <svg xmlns="http://www.w3.org/2000/svg" width="128" height="128" fill="currentColor"
+                            class="bi bi-person" viewBox="0 0 16 16">
+                            <path stroke="white" stroke-width="0.5px"
+                                d="M8 8a3 3 0 1 0 0-6 3 3 0 0 0 0 6zm2-3a2 2 0 1 1-4 0 2 2 0 0 1 4 0zm4 8c0 1-1 1-1 1H3s-1 0-1-1 1-4 6-4 6 3 6 4zm-1-.004c-.001-.246-.154-.986-.832-1.664C11.516 10.68 10.289 10 8 10c-2.29 0-3.516.68-4.168 1.332-.678.678-.83 1.418-.832 1.664h10z" />
+                        </svg>
+                    </div>
+                </div>
+            </div>
+
+            <!-- explanation text -->
+            <div class="esplanation">
+                <p class="pEsplanation">
+                    Lorem ipsum dolor, sit amet consectetur adipisicing elit. Atque nostrum libero quibusdam similique
+                    maxime
+                    optio suscipit sapiente? Eius ea eum, saepe, accusantium recusandae dolore optio repudiandae ad est
+                    aperiam
+                    quas? Lorem ipsum dolor, sit amet consectetur adipisicing elit. Atque nostrum libero quibusdam
+                    similique
+                    maxime
+                    optio suscipit sapiente? Eius ea eum, saepe, accusantium recusandae dolore optio repudiandae ad est
+                    aperiam
+                    quas?
+                </p>
+            </div>
+        </div>
+
+        <form>
```

Comment:
I am wondering whether this is a separate submission form or a duplicate of the form with the id `generateTrack` above.